clang/docs/ReleaseNotes.rst

```diff
@@ -854,9 +854,15 @@ and `-mbulk-memory` flags, which correspond to the [Bulk Memory Operations]
 and [Non-trapping float-to-int Conversions] language features, which are
 [widely implemented in engines].
 
+A new Lime1 target CPU is added, -mcpu=lime1. This CPU follows the definition of
+the Lime1 CPU [here], and enables -mmultivalue, -mmutable-globals,
+-mcall-indirect-overlong, -msign-ext, -mbulk-memory-opt, -mnontrapping-fptoint,
```

Comment:
What is the rationale for including the overlong call indirect here? 

Would it be unreasonable to suggest that we land the two new features in a separate PR before landing the new CPU type that uses them?  (Maybe its hard to split up and test?)

Reply:
I've now split out those parts into https://github.com/llvm/llvm-project/pull/117087.